--- percona-xtrabackup-2.3.3/./storage/innobase/xtrabackup/src/CMakeLists.txt~	2015-12-14 21:49:07.000000000 +0200
+++ percona-xtrabackup-2.3.3/./storage/innobase/xtrabackup/src/CMakeLists.txt	2016-01-02 01:20:57.242088388 +0200
@@ -18,7 +18,6 @@
 INCLUDE(libev)
 
 ADD_SUBDIRECTORY(libarchive)
-ADD_SUBDIRECTORY(jsmn)
 
 FIND_GCRYPT()
 FIND_CURL()
@@ -37,7 +36,6 @@
   ${CMAKE_SOURCE_DIR}/sql
   ${CMAKE_SOURCE_DIR}/storage/innobase/xtrabackup/src/libarchive/libarchive
   ${CMAKE_SOURCE_DIR}/storage/innobase/xtrabackup/src/quicklz
-  ${CMAKE_SOURCE_DIR}/storage/innobase/xtrabackup/src/jsmn
   ${GCRYPT_INCLUDE_DIR}
   ${CURL_INCLUDE_DIRS}
   ${LIBEV_INCLUDE_DIRS}
